courses/B.yaml

```diff
@@ -4,6 +4,8 @@
     becomes unusable.  There are good and bad ways to do this.
   reading: |
     `Aalto scicomp data docs <https://scicomp.aalto.fi/data/>`__
+  video: |
+    `Research Data Management 101 <https://www.youtube.com/watch?v=_pHVMwfOZf8>`_
```

Comment:
```suggestion
    `What is not data management, in 5 minutes <https://www.youtube.com/watch?v=N2zK3sAtr-4>`_
    `Research Data Management 101 <https://www.youtube.com/watch?v=_pHVMwfOZf8>`_
```
This is one of my favorite ways to show what is data management - by showing the opposite of it.  Very relateable that way.